app/components/handle-input/component.js

```diff
@@ -36,6 +36,9 @@ export default FocusableInput.extend({
     },
     set(key, value) {
       this._sanitizedValue = sanitizeInput(value);
+
+      this.attrs.value.update(this._sanitizedValue);
```

Comment:
Interesting that `get()` / `set()` didn't have to be removed in favor of `didReceiveAttrs()` or another lifecycle hook.